starter/README.md

```diff
@@ -6,7 +6,115 @@ This is the project template for Stark applications.
+|   |   \   translation.config.ts       # helper functions for the translations
+|   |
+|   +---assets                          # static assets (fonts, images, translations, ...)
+|   |   |
+|   |   +---fonts                       # application-specific fonts
+|   |   |   \
+|   |   |
+|   |   +---images                      # application-specific images
+|   |   |   |
+|   |   |   +---app-icons               # application icons for mobile devices
+|   |   |   |   \
+|   |   |   |
+|   |   |   +---touch                   # touch icons for mobile devices
+|   |   |   |   \
+|   |   |   \
+|   |   |
+|   |   +---mock-data                   #
+|   |   |   \
+|   |   |
+|   |   +---translations                # application-specific translations
+|   |   |   |                           #
+|   |   |   |   en.json                 # English
+|   |   |   |   fr.json                 # French
+|   |   |   \   nl.json                 # Dutch
+|   |   |   
+|   |   \   README.md                   # 
+|   |
+|   +---assets-base                     # static assets that will be copied to the root of the application
+|   |   |                               #
+|   |   |   browserconfig.xml           # application icons for Windows mobile devices
+|   |   |   crossdomain.xml             # cross domain policies
+|   |   |   favicon.ico                 # icon for bookmarks bar
+|   |   |   humans.txt                  # contains information about the website (http://humanstxt.org/)
+|   |   |   manifest.json               # application icons for Android mobile devices
+|   |   |   README.md                   # 
+|   |   |   robots.txt                  # the robots exclusion protocol (http://www.robotstxt.org/)
+|   |   \   service-worker.js           # support for building Progressive Web Applications (PWA) with Service Workers 
+|   |
+|   +---environments                    # configuration variables for each environment
+|   |   |                               # 
+|   |   |   environment.e2e.prod.ts     # e2e tests configuration
+|   |   |   environment.hmr.ts          # development with HMR (Hot Module Replacement) configuration
+|   |   |   environment.prod.ts         # production configuration
+|   |   |   environment.ts              # development configuration
+|   |   \   model.ts                    # interface defining the variables
+|   |
+|   +---styles                          # application-specific styles
+|   |   \
+|   |
+|   |   custom-typings.d.ts             # add your own type definitions that can't be found in the registry in this file
+|   |   hmr.ts                          # Hot Module Replacement logic
+|   |   index.html                      # the main HTML page that is served when someone visits this site
+|   |   main.browser.ts                 # Angular bootstrapping
+|   |   polyfills.browser.ts            # this file includes polyfills needed by Angular and is loaded before the app
```

Comment:
Perhaps we can say "should be loaded before the app"... so they know that is important